### PR TITLE
doc/getting-started: add null_blk tutorial

### PR DESCRIPTION
**Pull request by zdover23**
This PR adds a tutorial that will guide first-time
readers to create a zoned null block device.

This is the simplest of the three scenarios that
we will add to the guide. The other two, in order
of increasing complexity, are (1) emulated devices
(either qemu ZNS or scsi tcmu-runner) and (2) real
hardware.

This is part of an initiative to reorganize the Getting
Started page so that it turns beginners into adepts as
quickly as possible.

Signed-off-by: Zac Dover <zac.dover@gmail.com>

### CODE REVIEW

File: docs/getting-started/index.md [hunk 1]
```diff
@@ -7,8 +7,30 @@ slug: /getting-started
 
 # Getting Started
 
+## Overview
+
 Learn how to set up a Linux system to use zoned storage devices.
 
+## Introduction
+
+If you have never used a Shingled Magnetic Recording (SMR) disk before, this is
```
Comment (damien-lemoal) [inline docs/getting-started/index.md:16]: Let's be more general here: replace "Shingled Magnetic Recording (SMR) disk" with "zoned block device". And you can make that a link to https://zonedstorage.io/docs/introduction/zoned-storage.

File: docs/getting-started/index.md [hunk 2]
```diff
@@ -7,8 +7,30 @@ slug: /getting-started
...
+
+The following three-step procedure results in an environment that will allow
+you to evaluate ZonedStorage and to investigate its features. This is the
+fastest way to get started, and is recommended for first-time readers.
+
+This procedure creates a zoned null block device. It is okay if you do not yet
+know what that means. This is the simplest way to try out ZonedStorage.
+
+1. Install a Linux distribution that supports ZonedStorage. Fedora 36 is recommended.
+1. Install ``lsblk`` and ``blkzone`` on the Linux distribution that supports ZonedStorage. In Fedora 36, the package called ``util-linux`` provides ``lsblk`` and ``blkzone``.
```
Comment (damien-lemoal) [inline docs/getting-started/index.md:29]: There is no need to install lsblk and blkzone. These are part of util-linux sys-utils which is installed by default, always, on any Linux distro.

Comment (zdover23) [reply docs/getting-started/index.md:29]: Roger that. I've removed it.

Comment (damien-lemoal) [reply docs/getting-started/index.md]: The text you have is OK and simple, but I do not like very much how the getting started button landing page now looks. It is messy... We need to think how to better integrate this, or redesign the getting started landing page.
Let me try a few things (I have pulled locally your patch, through git cli).

Comment (zdover23) [reply docs/getting-started/index.md]: My plan was to get these tutorial sections into the documentation and then streamline the landing page.

**Pull request closed without merging**